queries.yaml

```diff
@@ -821,12 +821,20 @@ post_process:
       start_zoom: 0
       end_zoom: 14
       properties:
-        - all_bicycle_networks
-        - all_bicycle_shield_texts
         - all_walking_networks
         - all_walking_shield_texts
         - all_bus_networks
         - all_bus_shield_texts
+  # drop all_bicycle_* until the max zoom:
+  # https://github.com/tilezen/vector-datasource/pull/1707#discussion_r236522176
```

Comment:
Ah, I didn't expand the diff enough in the earlier review. It's probably better to be more consistent and just drop all the `all*` in the same zoom range. I'd be fine making that end zoom 15 and call it good?

Reply:
It's a bit of a corner-case thing, since we only export `all_*` properties on JSON tiles. I like them for multi-shielding, but AFAIK, no one is using it for that and it does affect merging... So perhaps this is something that we come back to once we're into multi-shielding territory?

Reply:
Sure